src/GameLogic/GameSession.ts

```diff
@@ -38,6 +38,19 @@ export class GameSession {
 		this._recordings.push(actionSequence);
 	}
 
+	public removeRecording(actionSequence: ActionSequence): void {
+		const index = this._recordings.indexOf(actionSequence);
+		if (index === -1) {
+			throw new Error('Attempted to remove a recording that does not exist');
+		}
+
+		this._recordings.splice(index, 1);
+	}
+
+	public getRecordings(): readonly ActionSequence[] {
```

Comment:
Should this be `public get recordings()`? Or are we making it a method because it returns a readonly value?

Reply:
It'd probably make more sense to make it a getter. I think the reason I made it a method was because I thought it'd be easier to refactor out into its own storage later (which is a lie, it'll be as easy/difficult either way). Let me fix it!